types/parse/index.d.ts

```diff
@@ -727,6 +727,15 @@ declare global {
         }
         const Installation: InstallationConstructor;
 
+        // Read preference describes how MongoDB driver route read operations to the members of a replica set.
+        enum ReadPreferenceOption {
```

Comment:
cannot be enum unless there exists the object with this name in the package to be used.

Reply:
Also documentation you pointed shows function parameters of type string and not special string types so this change looks incorrect